src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,120 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Scanner;
+
+import duke.domain.Task;
+
+/**
+ * Storage class in charge of storing the tasks in a text file.
+ */
+public class Storage {
+
+    private static final String delimiter = "@@@";
+
+    /**
+     * Ensures that the data file exists, then reads the data file and returns a
+     * list of tasks
+     *
+     * @param filePath
+     *            The path to the file where the data is stored.
+     * @return A list of tasks
+     */
+    public List<Task> load(String filePath) {
+        try {
+            ensureDataFileExist(filePath);
+            List<Task> initTask = new ArrayList<>();
```

Comment:
Since this is a list of Task objects, the variable name should be "initTasks" instead. :)